stack/cluster.py

```diff
@@ -139,9 +139,9 @@
 
 
 template.add_mapping("ECSRegionMap", {
-    "eu-west-1": {"AMI": "ami-4e6ffe3d"},
-    "us-east-1": {"AMI": "ami-8f7687e2"},
-    "us-west-2": {"AMI": "ami-84b44de4"},
+    "eu-west-1": {"AMI": "ami-a1491ad2"},
+    "us-east-1": {"AMI": "ami-eca289fb"},
+    "us-west-2": {"AMI": "ami-9fadf8ff"},
```

Comment:
Good catch! I wonder if we should just make this a text field? What do you think?